Exercise_A/index.html

```diff
@@ -1,45 +1,190 @@
+        result on google, edit a few lines, and you're on your way! No one will notice!</p>
+      <button type="button" class="btn btn-light rounded-pill">GOOGLE THAT SHIT</button>
+    </div>
+  </div>
+  <div id="fouricons" class="row d-flex justify-content-center">
+    <div class="col text-center w-75 mx-auto p-5">
+      <h2>So Fucking Creative</h2>
+      <hr>
+    </div>
+    <div class="d-inline-flex align-items-stretch p-5 flex-wrap">
+      <div class="col">
+        <div class="card text-center">
+          <i class="fas fa-globe-asia fa-5x p-2"></i>
+          <div class="card-body">
+            <h5 class="card-title">This Fucking Template</h5>
+            <p class="card-text">Guaranteed to use the same fucking template that every other bootstrap website uses,
+              downloaded straight from The Web™</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card text-center">
+          <i class="fas fa-paper-plane fa-5x p-2"></i>
+          <div class="card-body">
+            <h5 class="card-title">This Set Of Four Icons</h5>
+            <p class="card-text">Look at this cool set of four icons describing different things about us! We use
+              four, because it's the default.</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card text-center">
+          <i class="fas fa-thumbs-up fa-5x p-2"></i>
+          <div class="card-body">
+            <h5 class="card-title">Lots Of Effort</h5>
+            <p class="card-text">We even changed some of the icons! We take the extra effort to make our designs truly
+              original.</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card text-center">
+          <i class="fas fa-heart fa-5x p-2"></i>
+          <div class="card-body">
+            <h5 class="card-title">Made with Love</h5>
+            <p class="card-text">Because nothing says hard work and talent like editing a few lines of text.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div id="bullshit" class="row">
+    <div class="d-flex">
+      <div class="col">
+        <div class="card bg-dark text-white">
+          <img class="card-img rounded-0" src="assets/images/1.jpg" alt="Card image">
+          <div class="card-img-overlay text-center bg-dark rounded-0">
+            <h5 class="card-title">One Thing</h5>
+            <p class="card-text">Look</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card bg-dark text-white">
+          <img class="card-img rounded-0" src="assets/images/2.jpg" alt="Card image">
+          <div class="card-img-overlay text-center bg-dark rounded-0">
+            <h5 class="card-title">Two Thing</h5>
+            <p class="card-text">At</p>
+          </div>
+        </div>

```

Comment:
Your column system works a little but could be better.
Add in breakpoints and how many columns you want while the page is that size.
Come back and alter this section and the section above